### PR TITLE
Fix webcomponents-lite bundling 

### PR DESCRIPTION
**Pull request by grafixeyehero**
i don't know why firefox not working. but this well fix.

### CODE REVIEW

File: src/bundle.js [hunk 1]
```diff
@@ -4,12 +4,6 @@
 // Use define from require.js not webpack's define
 var _define = window.define;
 
-// document-register-element
-var docRegister = require("document-register-element");
-_define("document-register-element", function() {
-    return docRegister;
-});
-
```
Comment (JustAMan) [inline src/bundle.js]: why remove this and other piece?

**Comment by dkanada** [comment]
@dmitrylyzo is document-register-element also causing issues or is it just the webcomponents library?

**Comment by dmitrylyzo** [comment]
> @dmitrylyzo is document-register-element also causing issues or is it just the webcomponents library?

`document-register-element` seems good. At least, there are no errors in the console, and Firefox is loading.

File: src/bundle.js (hunk 1 above)
Comment (grafixeyehero) [reply src/bundle.js]: i don't know why this happening on firefox. removeing form bundle fix the problem.

Comment (dmitrylyzo) [reply src/bundle.js]: :confused: For me, only `webcomponents` caused an issue in Firefox.

**Comment by dkanada** [comment]
My charset pull request does fix another console error on Firefox so we should merge that one as well.

File: src/bundle.js (hunk 1 above)
Comment (grafixeyehero) [reply src/bundle.js]: ahh i have found the way to fix

**Comment by dkanada** [comment]
> Module not found: Error: Can't resolve 'webcomponents.js/webcomponents-lite'

The build is failing for me on this branch. How about using the dependency @dmitrylyzo mentioned in [this](https://github.com/jellyfin/jellyfin-web/pull/597#issuecomment-558779051) comment?

**Comment by dmitrylyzo** [comment]
> The build is failing for me on this branch.

I tried `fix` branch (+merged `master` to it) and have no errors (`npm update` + `npm run build`).
Resulting `bundle.js` has `webcomponents-lite` code.
Firefox loaded.
IE11 fails on `bundle.js` (on `class` keyword I suppose) - according to `site.js`, `webcomponents` is for `msie`.

> How about using the dependency @dmitrylyzo mentioned in [this](https://github.com/jellyfin/jellyfin-web/pull/597#issuecomment-558779051) comment?

Mentioned by me is older.

**Comment by grafixeyehero** [comment]
Firefox load correctly form my side.

**Pull request merged**